.storybook/preview.js

```diff
@@ -30,27 +19,13 @@ export const globalTypes = {
       items: Object.keys(theme.colorSchemes),
       showName: true
     }
-  },
-  nightScheme: {
-    name: 'Night color scheme',
-    description: 'Night color scheme',
-    defaultValue: 'dark',
-    toolbar: {
-      icon: 'circle',
-      items: Object.keys(theme.colorSchemes),
-      showName: true
-    }
   }
 }
 
 // context.globals.X references items in globalTypes
 const withThemeProvider = (Story, context) => {
   return (
-    <ThemeProvider
-      colorMode={context.globals.colorMode}
-      dayScheme={context.globals.dayScheme}
-      nightScheme={context.globals.nightScheme}
-    >
+    <ThemeProvider colorMode={'day'} dayScheme={context.globals.colorMode} nightScheme={'dark'}>
```

Comment:
We don't have to define nightScheme here since `dark` is the default value.

```suggestion
    <ThemeProvider colorMode="day" dayScheme={context.globals.colorScheme}>
```